sass/true/_modules.scss

```diff
@@ -65,57 +66,49 @@
 ///         (1px solid red, 1px dashed green, 3px solid blue));
 ///     }
 ///   }
-@mixin describe(
-  $name
-) {
+@mixin describe($name) {
   @include test-module($name) {
     @content;
   }
 }
 
-
-
 // Module Start
 // ------------
 /// Module start helper
 /// @access private
 /// @group private-test
 /// @param {string} $name - module name
-@mixin _true-module-start(
-  $name
-) {
-  @include _true-context(module, $name);
-  $module-list: _true-context-all('module');
-  $modules: nth($module-list, 1);
+@mixin module-start($name) {
+  @include context.context(module, $name);
+  $module-list: context.context-all('module');
+  $modules: list.nth($module-list, 1);
 
-  $length: length($module-list);
+  $length: list.length($module-list);
 
   @if $length > 1 {
     @for $i from 2 through $length {
-      $modules: $modules + ' :: ' + nth($module-list, $i);
+      $modules: $modules + ' :: ' + list.nth($module-list, $i);
     }
   }
 
-  @include _true-message('# Module: #{$modules}', 'comments');
+  @include messages.message('# Module: #{$modules}', 'comments');
 
-  $underline: '----------';
+  $underline: '--------';
```

Comment:
@mirisuzanne I'm, uh, not sure why this change was necessary to maintain the right length of underlines?